packages/react-native-editor/ios/Podfile

```diff
@@ -1,10 +1,14 @@
 project 'GutenbergDemo.xcodeproj'
 
-require_relative '../../../node_modules/react-native/scripts/react_native_pods'
-require_relative '../../../node_modules/@react-native-community/cli-platform-ios/native_modules'
+# Resolve react_native_pods.rb with node to allow for hoisting
+require Pod::Executable.execute_command('node', ['-p',
+  'require.resolve(
+    "react-native/scripts/react_native_pods.rb",
+    {paths: [process.argv[1]]},
+  )', __dir__]).strip
 
 # Uncomment the next line to define a global platform for your project
-app_ios_deployment_target = Gem::Version.new('13.0')
+app_ios_deployment_target = Gem::Version.new('13.4')
```

Comment:
Noting the deployment target bump here done to address an error during `pod install`. We should consider if this bump is a problem here in the Demo editor or in the host app.